keepassxc-browser/_locales/en/messages.json

```diff
@@ -811,6 +815,10 @@
         "message": "Show notifications for errors and when user interaction is required.",
         "description": "Show notifications option help text."
     },
+    "optionsUseMonochromeToolbarIconHelpText": {
+        "message": "Use a monochrome icon with colored badges in toolbar that adapts to light/dark browser theme. Requires browser restart.",
```

Comment:
One final fix to this: at least for me browser restart is no longer required. If the system theme is used, the icon will change after a tab is switched. With a specified dark/light theme the icons change instantly. So maybe this part of the text could be removed.

Reply:
Oh, true! Seems like when "System" theme is enabled, a restart is not necessary. However, on my system it still required an _interaction_ with the icon (e.g. hover or click) to change it from dark/ligh to colored or vice versa.

How about I change it to "(Might require browser restart)"?

Reply:
Just added "might" here: [d9c0082](https://github.com/keepassxreboot/keepassxc-browser/pull/1938/commits/d9c008209faea6b16978e411f8b4550faf1982ad)